install.sh

```diff
@@ -45,10 +45,19 @@ execute() {
   hash_sha256_verify "${tmp_dir}/${TARBALL}" "${tmp_dir}/${CHECKSUM}"
 
   (cd "${tmp_dir}" && untar "${TARBALL}")
-  install -d "${BINDIR}"
-  install "${tmp_dir}/${NAME}/${BINARY}" "${BINDIR}/"
+
+  if [ -w "$BINDIR" ]; then
```

Comment:
Do you mind adding a comment above with what the conditional does? Otherwise everyone will need to google what `-w` does to understand this 😅

Reply:
Done 👍🏼